lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -507,6 +496,46 @@ def sectionNameFromSchema(self, schemaTypeName):
 
         return schemaTypeName
 
+    def createShaderAttributesSection(self):
```

Comment:
Move all shader parameters to top of the AE. They used to reside in "Extra Attributes".